core-it-suite/src/test/java/org/apache/maven/it/MavenITmng7568SettingsProfileDeactivationTest.java

```diff
@@ -0,0 +1,65 @@
+package org.apache.maven.it;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import org.apache.maven.it.util.ResourceExtractor;
+
+import java.io.File;
+
+public class MavenITmng7568SettingsProfileDeactivationTest
+        extends AbstractMavenIntegrationTestCase
+{
+    private static final String PROJECT_PATH = "/mng-7568-inactive-profile-deactivation";
+
+    public MavenITmng7568SettingsProfileDeactivationTest()
+    {
+        super( "[3.0.4,4)" );
```

Comment:
I think we should be able to keep this test working for Maven 4, right? I don't think we covered this flow in any other ITs yet, so it's valuable to keep :)

Reply:
Not sure it'd do any good, as in master (4) there is no "could not be activated because it does not exist" in code base at all. It'd be insane to check that it's not present as there is no way for it to be...